spec/index.bs

```diff
@@ -10233,6 +10252,16 @@ dictionary GPURenderPassDescriptor : GPUObjectDescriptorBase {
     of the attachments are the same, so we can take any of them, assuming the descriptor is valid.
 </div>
 
+<div algorithm>
+    <dfn abstract-op>Validating GPURenderPassDescriptor's total color attachment depth</dfn>({{GPUDevice}} |device|, sequence<{{GPURenderPassColorAttachment}}?> |colorAttachments|)
+
+    1. Let |formats| be an empty sequence<{{GPUTextureFormat}}>
+    1. For each |colorAttachment| in |colorAttachments|:
+        1. If |colorAttachment| is {{undefined}}, continue.
```

Comment:
nit: `{{undefined}}` links to the type, not the value. I don't think there's a way to link to the value so I just use `` `undefined` ``.